src/libraries/Microsoft.Extensions.DependencyModel/src/DependencyContextJsonReader.cs

```diff
@@ -209,16 +210,16 @@ private DependencyContext Read(Utf8JsonReader reader)
             }
 
             return new DependencyContext(
-                new TargetInfo(framework, runtime, runtimeSignature, isPortable),
+                new TargetInfo(framework, runtime, runtimeSignature ?? string.Empty, isPortable),
```

Comment:
Why this change? I believe `TargetInfo.RuntimeSignature` can be `null`.

Reply:
It's assigned empty string in tests, so I thought, it's not meant to be `null`.

![image](https://user-images.githubusercontent.com/6609929/129685827-bb70028f-a6c8-420c-aadd-2cbac0cf2ebe.png)
![image](https://user-images.githubusercontent.com/6609929/129685875-c2943023-6fd4-4e30-863f-2d21090149a6.png)

Reply:
I don't think we should be changing behavior here. Please keep the library's behavior the same.